qiita_plugins/target_gene/tgp/pick_otus.py

```diff
@@ -155,17 +176,25 @@ def pick_closed_reference_otus(qclient, job_id, parameters, out_dir):
         raise ValueError(error_msg)
     reference_fps = ref_info['filepaths']
 
-    qclient.update_job_step(job_id, "Step 2 of 3: Generating command")
+    qclient.update_job_step(job_id, "Step 2 of 4: Generating command")
     command, pick_out = generate_pick_closed_reference_otus_cmd(
         fps, out_dir, parameters, reference_fps)
 
-    qclient.update_job_step(job_id, "Step 3 of 3: Executing OTU picking")
+    qclient.update_job_step(job_id, "Step 3 of 4: Executing OTU picking")
     std_out, std_err, return_value = system_call(command)
     if return_value != 0:
         error_msg = ("Error running OTU picking:\nStd out: %s\nStd err: %s"
                      % (std_out, std_err))
         return format_payload(False, error_msg=error_msg)
 
+    qclient.update_job_step(job_id, "Step 4 of 4: tgz sortmerna folder")
```

Comment:
`"Step 4 of 4: tgz sortmerna folder"` -> `"Step 4 of 4: Generating tgz sortmerna folder"`